webapp/src/ts/providers/parse.provider.ts

```diff
@@ -272,12 +282,16 @@ class ASTCompiler {
     const ast = this.cAst;
     const stmts = this.cStmts;
     const _args = [];
+
     for (const arg of ast.args) {
       _args.push(this.build(arg));
     }
-    const fn = this.build(ast.receiver);
+
+    const functionName = ast.receiver.name;
+    const parent = this.build(ast.receiver.receiver);
```

Comment:
`parent` is already used in browsers, as a global object in the execution environment. I'd prefer if we don't reuse this variable name.

Reply:
🤦🏽‍♀️ I missed that, I'll rename the variable. Thanks

Reply:
Internet Explorer used to throw errors 15 years ago if you overwrote `parent` :D

Reply:
No worries. I bet we overwrite it in other places in webapp code. I have flashbacks from the old days when I see it assigned to something else.

Reply:
Done